tests/test_other.py

```diff
@@ -4630,7 +4630,7 @@ def test_only_force_stdlibs_2(self):
   }
 }
 ''')
-    with env_modify({'EMCC_FORCE_STDLIBS': 'libc,libc++abi,libc++,libdlmalloc,libpthreads_stub', 'EMCC_ONLY_FORCED_STDLIBS': '1'}):
+    with env_modify({'EMCC_FORCE_STDLIBS': 'libc,libc++abi,libc++,libmalloc,libpthreads', 'EMCC_ONLY_FORCED_STDLIBS': '1'}):
```

Comment:
what is `libmalloc`? in embuilder.py help I see `libdlmalloc` as I'd expect. are the names not coordinated between the two places?

also, why is `pthreads_stub` changing to `pthreads` here? that's a different library, or did the meaning change?

Reply:
In the new system, we use simple names to refer to libraries. Depending on the current setting, it will link in the correct version of the library.

For example, `libmalloc` links in `dlmalloc` or `emmalloc` depending on what's specified in `Settings`. Similarly, `libpthreads` links in the actual library or the stub version.